website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -261,19 +263,20 @@ spec:
 
 </CodeBlockConfig>
 
--> **NOTE**: If TLS is enabled for external services registered through the Consul catalog, you must include the `caFile` parameter that points to the system trust store of the terminating gateway container.
+-> **NOTE**: If TLS is enabled for external services registered through the Consul catalog and you are not using [transparent proxy `Destinations`](#register-an-external-service-as-a-destination), you must include the `caFile` parameter that points to the system trust store of the terminating gateway container.
```

Comment:
What `caFile` parameter is this talking about? I know that this was here before your change but maybe we can link to the param? It's confusing reading this what it's talking about.